pkg/make/builder.go

```diff
@@ -124,8 +124,8 @@ func (b *Builder) ExtBinary(input string) string {
 	}
 }
 
-// PlatformBinary is a helper to return the platform specific binary suffix.
-func (b *Builder) PlatformBinary(input string) string {
+// BinExtension is a helper to return the platform specific binary suffix.
+func (b *Builder) BinExtension(input string) string {
```

Comment:
BinExtension immediately made me think this was for making the osquery extension binary. Would you consider calling this BuildFileName, BinaryPath, etc?

Reply:
Sure. These helpers are now `PlatformExtensionName` and `PlatformBinaryName`